sycl/plugins/unified_runtime/ur/adapters/level_zero/event.cpp

```diff
@@ -1094,24 +1094,29 @@ ur_result_t _ur_ze_event_list_t::createAndRetainUrZeEventList(
 
     if (EventListLength > 0) {
       for (uint32_t I = 0; I < EventListLength; I++) {
+        auto Queue = EventList[I]->UrQueue;
         {
           std::shared_lock<ur_shared_mutex> Lock(EventList[I]->Mutex);
           if (EventList[I]->Completed)
             continue;
 
-          // Poll of the host-visible events.
-          auto HostVisibleEvent = EventList[I]->HostVisibleEvent;
-          if (FilterEventWaitList && HostVisibleEvent) {
```

Comment:
Hmm. I made the default setting to not do any filtering in the plugin in https://github.com/intel/llvm/commit/dbde93f8eb96d6c7b29e2386aac7a7cd64aef84d

This somehow got lost with the move to adapter.
With it I think this change is not necessary.

Reply:
Thanks @smaslov-intel . Yes, it seems I missed this change https://github.com/intel/llvm/commit/dbde93f8eb96d6c7b29e2386aac7a7cd64aef84d

```cpp
const bool RetVal = Ret ? std::stoi(Ret) : 0;
```

I will do some testing on this.